README.md

```diff
@@ -1,68 +1,53 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Coding to Learn and Create (C2LC) Interface
```

Comment:
I really like that idea of including the word inclusive!

Reply:
Both Simon and Michelle liked the idea of using the word inclusive, but they want to remove C2LC from the title, since people won't understand what that stands for.

Reply:
I agree.